index.html

```diff
@@ -41,20 +44,155 @@ <h1 id="nm">I’m Mariane<br><br>
+                        <ul>
+                            <li>&bull;&nbsp;&nbsp; Back End Dev</li>
+                            <li>&bull;&nbsp;&nbsp; 2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                    <ul class="tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li class="js">javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div>
+                    <img src="./assets/Snapshoot Portfolio (2).png" alt="">
+                    <h2>Multi-Post<br>
+                        Stories</h2>
+                    <div class="details">
+                        <h3><b>CANOPY</b></h3>
+                        <ul>
+                            <li>&bull;&nbsp;&nbsp; Back End Dev</li>
+                            <li>&bull;&nbsp;&nbsp; 2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                    <ul class="tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li class="js">javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="about">
+            <div>
+                <h2>About<br>Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                    of my work and experience! If you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.</p>
+                <p class="lnk2">LET’S CONNECT</p>
+                <div class="socials2">
+                    <ul>
+                        <li><a href="https://www.linkedin.com/in/oussama-elabdioui-4677a41b6/"><img
+                                    src="./assets/github.png" alt=""></a></li>
+                        <li><a href="https://github.com/codedit334"><img src="./assets/Linkedin.png" alt=""></a></li>
+                        <li><a href="#"><img src="./assets/peace.png" alt=""></a></li>
+                        <li><a href="https://twitter.com/oussedit"><img src="./assets/twiter.png" alt=""></a></li>
+                        <li><a href="#"><img src="./assets/M.png" alt=""></a></li>
+
+                    </ul>
+                </div>
+                <button>See my resume</button>
+
+                <div class="languages">
+                    <span class="aboutMenu">
+                        <h2>Languages</h2>
+                        <img src="./assets/down.png" width="38px" alt="">
```

Comment:
[OPTIONAL] Good job adding the arrow as requested by the previous reviewer 👍. To make the UI for this section much nicer, I suggest reducing the sizes of each arrow.